service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/TemporaryHearingJudgeContentProvider.java

```diff
@@ -20,9 +21,23 @@
 
     public TemporaryHearingJudgeTemplate buildNotificationParameters(CaseData caseData, HearingBooking hearingBooking) {
         JudgeAndLegalAdvisor judgeAndLegalAdvisor = hearingBooking.getJudgeAndLegalAdvisor();
+
+        String judgeTitle = judgeAndLegalAdvisor.getJudgeOrMagistrateTitle();
+        String judgeName = judgeAndLegalAdvisor.getJudgeName();
+
+        if (judgeAndLegalAdvisor.getJudgeTitle().equals(MAGISTRATES)) {
+            if (judgeName != null) {
```

Comment:
I don't think it is possible but can this be empty?

Reply:
pretty sure it can't, getJudgeName() will grab judgeFullName or judgeLastName